specutils/analysis/width.py

```diff
@@ -0,0 +1,190 @@
+import numpy as np
+from astropy.stats.funcs import gaussian_fwhm_to_sigma
+from ..spectra import SpectralRegion
+
+
+__all__ = ['sigma_full_width', 'full_width_half_max', 'equivalent_width']
+
+
+def _computation_wrapper(func, spectrum, region):
+
+    # No region, therefore whole spectrum.
+    if region is None:
+        return func(spectrum)
+
+    # Single region
+    elif isinstance(region, SpectralRegion):
+        return func(spectrum, region=region)
+
+    # List of regions
+    elif isinstance(region, list):
+        return [func(spectrum, region=reg) for reg in region]
+
+
+def sigma_full_width(spectrum, region=None):
```

Comment:
While it may seem a bit odd, I think *this* function is the  one that should use what you currently have for the FWHM.  But it should be called something like `gaussian_sigma_width`.  Using that convention, the current `full_width_half_max` could become `gaussian_fwhm` or similar.

(More on this in my review summary)